apps/gitness/src/styles.css

```diff
@@ -3,3 +3,11 @@
 @import '@harnessio/canary/src/styles.css'; */
 
 @import 'highlight.js/styles/atom-one-dark.css';
+
+:root {
+  --breadcrumbs-height: 55px;
+}
+
+.top-breadcrumbs {
```

Comment:
can we move this to a tailwind utility  instead of a custom class? it'll be more consistent